pkgs/applications/editors/abricotine/default.nix

```diff
@@ -0,0 +1,78 @@
+{ stdenv, lib, fetchurl, copyDesktopItems, makeDesktopItem, makeWrapper
+, writeText , electron }:
+
+let
+  # This script is mandatory to make this application work
+  # without user intervention. The reasoning is specified below:
+  # https://github.com/NixOS/nixpkgs/issues/148152#issuecomment-983773905
+  exec_script = writeText "exec_script" ''
```

Comment:
Could this be using `writeShellScriptBin`?

Reply:
Didn't know that one. Way more useful than `writeText`. Let me check.

Reply:
Ready, I adapted the setup and it works.